### ISSUE TITLE
RepositoryFiles.showRaw defines return type as Blob but returns string

### ISSUE THREAD

**Issue by WiseBird** [jdalrymple/gitbeaker#2276]
`showRaw` method is defined as follows:
```
export declare class RepositoryFiles<C extends boolean = false> extends BaseResource<C> {
    ...
    showRaw(projectId: string | number, filePath: string, options?: BaseRequestOptions): Promise<Blob>;
}
```
But as far as I can see in reality it returns `string` instead of `Blob`:

![image](https://user-images.githubusercontent.com/1745378/145845067-6d87f0f9-d409-4ffe-a5ec-359c626ccd6b.png)

Is it a wrong typing or I miss something?

**Comment by jdalrymple** [comment]
Sounds like i messed up the return type haha The processor in the backend encodes the response based on the content type. So even though its 'raw' (which is why its typed as a Blob), it could return a string, json or a blob. This is definitely a bug. Ill have to think of a way to enforce the return format.

Normally this isnt an issue because the return types are constant and consistent for each endpoint. This would be an exception.

**Comment by jdalrymple** [comment]
Update: it will return blob, the user will need to determine how they want to handle the blobbed response, either converting to string via .toString(), or writing to a file, in the case its an array buffer for example

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`37.0.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/37.0.0) :rocket:
